src/rfquack_config.h

```diff
@@ -23,8 +23,15 @@
 
 #include "config/general.h"
 #include "config/logging.h"
-#include "config/radio.h"
 #include "config/transport.h"
 #include "config/network.h"
 
+#ifndef RFQUACK_RADIO_MAX_MSG_LEN
+#define RFQUACK_RADIO_MAX_MSG_LEN 128
+#endif
+
+#ifndef RFQUACK_RADIO_TX_QUEUE_LEN
```

Comment:
This must be a left over, but please remove anything that has to do with *TX* queues; now we just send stuff directly to the radio. RX queues still make sense, because we might need to process packets before sending them to the host (e.g., filtering) or re-transmit them (e.g., packet modification).